docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md

```diff
@@ -0,0 +1,369 @@
+- **Specific version** option: Specify one or multiple existing versions of the process.
+
+Data from older versions is mapped to the most recent version in the selection. Therefore, the report data can seem to be inconsistent, which is due to changes that occurred within the diagram through the different versions. For example, the old versions do not contain newly added tasks or a part of the diagram was removed because it was considered to be obsolete.
+
+![Process definition selection for a multi-tenancy scenario](./img/tenantSelection.png)
+
+By default, all process instances for the selected process definitions are included in a report. You can reduce this set of process instances by applying a [filter](../../additional-features/filters.md).
+
+### Define the report
+
+In this section of the report builder, you are characterizing the output of the report. Basically, you are defining "I want to view ... grouped by ... visualized as ...". To understand better what "View" and "Group by" mean, you can use the analogy of a graph, where "View" is the y-axis and "Group by" is the x-axis.
+
+First, you need to select which part of the data you want to view. Optimize differentiates between the view (e.g. process instance or flow node) and the measure (e.g. count or duration):
+
+1. Raw Data: View just a table with the actual data listed as rows. This can come in handy if you found interesting insights in certain process instances and need detailed information about those instances, or you are exploring a process definition with a limited number of instances. This report type also allows you to inspect raw [object variable values](../../../../../self-managed/optimize-deployment/setup/object-variables.md).
+2. Process instance
+  * Count: View how many process instances were executed.
+  * Duration: View how long the process instances took to complete.
+3. Incident
+  * Count: View how many incidents occurred on the process.
+  * Resolution duration: View how long the incident took to get resolved.
+4. Flow node
+  * Count: View how often the flow nodes (e.g. tasks) have been executed.
+  * Duration: View how long each flow node took to complete.
+5. User task
+  * Count: View how often each user task has been executed.
+  * Duration: View how long each user task took to complete.
+6. Variable: View an aggregation of values for a specific numeric variable of the process definition.
+
+It is possible to display both count and duration measures for a single view in the same report.
+
+Subsequently, you need to define how to group the data. Think of it as applying a metric to your input, where you break up the data by date, flow nodes, variable values, or other properties. For that, you have different options:
+
+- None: Do not split up the data.
```

Comment:
I recommend formatting these options (none, flow nodes...) so they stand out more

Reply:
Adjusted in latest commit